### PR TITLE
Implement home screen design improvements

### PR DESCRIPTION
**Pull request by kubaolszewski**

### CODE REVIEW

File: lib/app/features/home/home_widgets/home_options_tap_bar/home_options_tap_bar.dart [hunk 1]
```diff
@@ -12,67 +12,44 @@ class HomeOptionsTapBar extends StatelessWidget {
...
-                  border: Border(
-                      bottom: BorderSide(
-                width: 3.0,
-                color: AppColors.mainThemePink,
-              ))),
-              child: Text(
-                context.localizations.homeUpcoming,
-                style: const TextStyle(color: Colors.white, fontSize: 16, fontWeight: FontWeight.bold),
+    return BlocBuilder<HomeCubit, HomeState>(
+      builder: (context, state) {
```
Comment (krzychudz) [inline lib/app/features/home/home_widgets/home_options_tap_bar/home_options_tap_bar.dart:17]: Consider adding buildWhen to rebuild only when necessary

File: lib/app/features/home/home_widgets/home_options_tap_bar/options_button_design.dart [hunk 2]
```diff
@@ -0,0 +1,34 @@
+
+import 'package:flutter/material.dart';
+import 'package:score_live/presentation/constants/app_colors.dart';
+
+class OptionsButtonDesign extends StatelessWidget {
```
Comment (krzychudz) [inline lib/app/features/home/home_widgets/home_options_tap_bar/options_button_design.dart]: Why not just **OptionButton**? **OptionsButtonDesign** sounds strange imo

Comment (kubaolszewski) [reply lib/app/features/home/home_widgets/home_options_tap_bar/options_button_design.dart]: applied, naming anything in code is surprisingly hard :sweat_smile:

File: lib/app/features/home/home_widgets/home_options_tap_bar/home_options_tap_bar.dart (hunk 1 above)
Comment (kubaolszewski) [reply lib/app/features/home/home_widgets/home_options_tap_bar/home_options_tap_bar.dart:17]: done

**Review by krzychudz — APPROVED** [review]

**Pull request merged**